src/Views/Form/Templates/Classic/resources/views/receipt.php

```diff
@@ -1,110 +1,125 @@
+				</p>
+				<div class="btn-row">
+					<button class="give-btn social-btn facebook-btn" onclick="GiveClassicTemplate.share(this);">
+						<?= esc_html__( 'Share on Facebook', 'give' ); ?>
+						<i class="fab fa-facebook"></i>
+					</button>
+					<button class="give-btn social-btn twitter-btn" onclick="GiveClassicTemplate.share(this);">
+						<?= esc_html__( 'Share on Twitter', 'give' ); ?>
+						<i class="fab fa-twitter"></i>
+					</button>
+				</div>
 			</div>
 		<?php endif; ?>
-		<h2 class="headline">
-			<?php echo $receipt->heading; ?>
-		</h2>
-		<p class="message">
-			<?php echo $receipt->message; ?>
-		</p>
-		<?php require 'social-sharing.php'; ?>
+
+		<div class="receipt-sections">
+
 		<?php
-		/* @var Section $section */
-		foreach ( $receipt as $section ) {
-			// Continue if section does not have line items.
-			if ( ! $section->getLineItems() ) {
-				continue;
-			}
-
-			if ( 'PDFReceipt' === $section->id ) {
-				$pdfReceiptLinkDetailItem = $section['receiptLink'];
-				continue;
-			}
-
-			echo '<div class="details">';
-			if ( $section->label ) {
-				printf( '<h3 class="headline">%1$s</h3>', $section->label );
-			}
-			echo '<div class="details-table">';
-
-			/* @var LineItem $lineItem */
-			foreach ( $section as $lineItem ) {
-				// Continue if line item does not have value.
-				if ( ! $lineItem->value ) {
+			/* @var Section $section */
+			foreach ( $receipt as $section ) :
+				if ( ! $section->getLineItems() || 'PDFReceipt' === $section->id )
 					continue;
-				}
-
-				// This class is required to highlight total donation amount in receipt.
-				$detailRowClass = '';
-				if ( DonationReceipt::DONATIONSECTIONID === $section->id ) {
-					$detailRowClass = 'totalAmount' === $lineItem->id ? ' total' : '';
-				}
-
-				printf(
-					'<div class="details-row%1$s">%2$s<div class="detail">%3$s</div><div class="value">%4$s</div></div>',
-					$detailRowClass,
-					$lineItem->icon,
-					$lineItem->label,
-					$lineItem->value
-				);
-			}
-			echo '</div>';
-			echo '</div>';
-		}
-		?>
-
-		<?php if ( $pdfReceiptLinkDetailItem ) : ?>
-			<div class="give-btn download-btn">
-				<?php echo $pdfReceiptLinkDetailItem->value; ?>
+			?>
+
+			<div class="details">
+				<?php if ( $section->label ): ?>
+					<h3 class="headline">
+						<?= $section->label; ?>
+					</h3>
+				<?php endif; ?>
+				<div class="details-table">
+
+					<?php
+						/* @var LineItem $lineItem */
+						foreach ( $section as $lineItem ):
+							if ( ! $lineItem->value )
+								continue;
+
+							$class = '';
+							if ( DonationReceipt::DONATIONSECTIONID === $section->id ) {
+								$class = 'totalAmount' === $lineItem->id ? 'total' : '';
+							}
+						?>
+
+						<div class="details-row <?= $class; ?>">
+							<?= $lineItem->icon; ?>
+							<div class="detail"><?= $lineItem->label; ?></div>
+							<div class="value"><?= $lineItem->value; ?></div>
+						</div>
+					<?php endforeach; ?>
+
+				</div>
```

Comment:
We can make this a definition list. The extra `div`s can stay since they are allowed to wrap `dt` and `dd` elements. I don’t think any CSS needs to change, but it might be good to double-check.

```suggestion
				<dl class="details-table">

					<?php
						/* @var LineItem $lineItem */
						foreach ( $section as $lineItem ):
							if ( ! $lineItem->value )
								continue;

							$class = '';
							if ( DonationReceipt::DONATIONSECTIONID === $section->id ) {
								$class = 'totalAmount' === $lineItem->id ? 'total' : '';
							}
						?>

						<div class="details-row <?= $class; ?>">
							<?= $lineItem->icon; ?>
							<dt class="detail"><?= $lineItem->label; ?></dt>
							<dd class="value"><?= $lineItem->value; ?></dd>
						</div>
					<?php endforeach; ?>

				</dl>
```